ToneMapper/TMGLFeatureManagerViewController.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015 Lightricks. All rights reserved.
+// Created by Barak Yoresh.
+
+#import <UIKit/UIKit.h>
+
+#import "TMFeature.h"
+
+@class TMGLEngine;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Generic Feature view controller enabling canceling and saving of feature parameters.
+@interface TMGLFeatureManagerViewController : UIViewController <TMFeatureDelegate>
+
+/// Setting super initializers to unavailable.
+- (instancetype)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
```

Comment:
Are you sure? because all my initialisers can return `nil` to be honest, and I'm not getting the "passing nil parameters to a nonnull blah blah" warning.